install_scripts/templates/kubernetes-upgrade.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+
+set -e
+AIRGAP=0
+
+{% include 'common/common.sh' %}
+{% include 'common/kubernetes.sh' %}
+{% include 'common/log.sh' %}
+{% include 'common/prompt.sh' %}
+{% include 'common/system.sh' %}
+
+#######################################
+# Upgrade Kubernetes on remote workers to master version
+# Globals:
+#   AIRGAP
+# Arguments:
+#   None
+# Returns:
+#   None
+#######################################
+upgradeK8sWorkers() {
+    masterVersion=$(getK8sMasterVersion)
+    version="v${masterVersion}"
+
+    workers=$(kubectl get nodes | sed '1d' | grep -v master | grep -v $version | awk '{ print $1 }')
+
+    for node in $workers; do
+        kubectl drain "$node" --ignore-daemonsets --delete-local-data --force
+        printf "\n\n\tRun the upgrade script on remote node before proceeding: ${GREEN}$node${NC}\n\n"
+        if [ "$AIRGAP" = "1" ]; then
+            printf "\t${GREEN}cat kubernetes-node-upgrade | sudo bash -s kubernetes-version=$version${NC}"
```

Comment:
I think these usually get a `.sh` when they're in the airgap bundle, right? so `kuberentes-node-upgrade.sh`?

Reply:
ok, right